README.md

```diff
@@ -136,6 +136,8 @@ See [here](https://github.com/Azure/azure-functions-core-tools/blob/master/READM
 
 * [Python 3.6.x](https://www.python.org/downloads/)
 
+> **NOTE**: [Python 3.6.8](https://www.python.org/downloads/release/python-368/) is the last release of Python 3.6 with installable binaries for Windows.
```

Comment:
@askpatrickw do you have any docs that say this will be the last release of Python 3.6.x? Is it assumed since they moved on to 3.7? Also do we know _why_ 3.6.9 doesn't have installable binaries?

Reply:
I don't know _why_ exactly,  but this is what is stated in the release notes at the link I included.
https://www.python.org/downloads/release/python-368/

Reply:
Good enough for me - thanks!